content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_etcd-healthcheck-client.md

```diff
@@ -1,98 +1,35 @@
 
-Generates the client certificate for liveness probes to healtcheck etcd
+Renew the certificate for liveness probes to healtcheck etcd
```

Comment:
Spotted a typo here. Best to fix it after release?

Reply:
I think so based on the time we have before release... I would suggest this be fixed after release.. @jimangel thoughts?

Reply:
I logged https://github.com/kubernetes/website/issues/14985